packages/site/src/components/ComponentDocExample.js

```diff
@@ -27,113 +29,65 @@ const styles = {
   })
 };
 
 const Root = createStyledComponent('div', styles.componentDocExample);
-const Resizable = createStyledComponent('div', styles.resizable);
 const H4 = createStyledComponent('h4', styles.h4);
-const Graf = createStyledComponent('p', styles.graf);
-
-function getDefaultValue(propDescription: Object): any {
-  const { defaultValue } = propDescription;
-  return (
-    (defaultValue && defaultValue.value && eval(defaultValue.value)) ||
-    undefined
-  );
-}
-
-function getFlowType(propDescription: Object): string {
-  const { flowType } = propDescription;
-  const type = (flowType && flowType.name) || 'unknown';
-  if (type === 'union') {
-    return `one of: ${flowType.raw}`;
-  }
-
-  return type;
-}
-
-function getExampleProps(
-  propDoc: Object = {},
-  propValues: Object = {}
-): Array<Object> {
-  return Object.keys(propDoc).sort().map(name => {
-    const propDescription = propDoc[name];
-    const { description, required } = propDescription;
-    const defaultValue = getDefaultValue(propDescription);
-    const exampleValue = propValues.hasOwnProperty(name)
-      ? propValues[name]
-      : defaultValue;
-    const type = getFlowType(propDescription);
-
-    return {
-      defaultValue,
-      description,
-      exampleValue,
-      name,
-      required,
-      type
-    };
-  });
-}
-
-function getComponentProps(exampleProps: Array<Object>): Object {
-  return exampleProps.reduce((acc, propDescription) => {
-    const propName = propDescription.name;
-    acc[propName] = propDescription.exampleValue;
-    return acc;
-  }, {});
-}
+const P = createStyledComponent('p', styles.p);
+const MyLivePreview = createStyledComponent(LivePreview, styles.livePreview, {
+  rootEl: 'div'
+});
+const MyLiveEditor = createStyledComponent(LiveEditor, styles.liveEditor);
 
 type Props = {
-  children?: MnrlReactNode,
+  backgroundColor?: string,
   className?: string,
-  component: MnrlReactComponent,
   description?: MnrlReactNode,
-  propDoc?: Object,
-  propValues?: Object,
-  source?: string,
+  hideSource?: boolean,
+  scope: Object,
+  source: string,
   title?: string
 };
 
 export default class ComponentDocExample extends Component {
   props: Props;
 
   render() {
-    const { className, component: Component, description, title } = this.props;
-    const componentProps = getComponentProps(
-      getExampleProps(this.props.propDoc, this.props.propValues)
-    );
+    const {
+      backgroundColor,
+      className,
+      description,
+      hideSource,
+      scope,
+      source,
+      title
+    } = this.props;
 
     return (
       <Root className={className}>
         <H4>{title}</H4>
-        {typeof description === 'string'
-          ? <Graf>{description}</Graf>
-          : description}
-        <Resizable>
-          <Component {...componentProps} />
-        </Resizable>
+        {typeof description === 'string' ? <P>{description}</P> : description}
+        <LiveProvider code={source} scope={scope}>
```

Comment:
The `<LiveProvider />` is injecting the same bunch of `prism` CSS for each example.  We should ideally set `mountStylesheet={false}` and then load the [react-live.css](https://github.com/FormidableLabs/react-live/blob/master/react-live.css) once for the whole site.  This should also have the effect of removing it from the snapshots.

This could be done as a followup story if you are anxious to get this deployed.